grails-app/services/au/org/ala/spatial/service/TasksService.groovy

```diff
@@ -208,8 +208,9 @@ class TasksService {
                 v.put('q', 'qid:' + qid)
             }
             else {
-                log.error("Failed to generate QID.")
-                throw new Exception("Error: failed to genereate qid!")
+                log.error("Failed to generate QID. Returned: " + qid)
+                log.debug(v.toString())
+                throw new Exception("Error: failed to generate qid!")
```

Comment:
For a task service, does this add the error to the task history and error lists?